app/models/label/LabelTable.scala

```diff
@@ -375,10 +376,34 @@ object LabelTable {
       "label_type_value" -> labelMetadata.labelTypeValue,
       "severity" -> labelMetadata.severity,
       "temporary" -> labelMetadata.temporary,
+      //this one is different because we're pulling from the tags and label_tags databases
+      "tags" -> getTagsFromLabelId(labelMetadata.labelId),
       "description" -> labelMetadata.description
     )
   }
 
+  //seperate sql request for getting tags, gets all the tags that correspond to the label Id.
+  def getTagsFromLabelId(labelId: Int): String = db.withSession { implicit session =>
+      val getTagsQuery = Q.query[Int, (String)](
+        """SELECT tag FROM sidewalk.tag
+             WHERE tag.tag_id IN (
+                  SELECT tag_id FROM sidewalk.label_tag
+                        WHERE label_tag.label_id = ?
+             )""".stripMargin
```

Comment:
Just trying to keep the indentation and such consistent, this is the indentation we are trying to use for raw SQL queries (I actually fix the indentation for a bunch of the queries in this file in a PR that is currently being reviewed):
```
"""SELECT tag
  |FROM sidewalk.tag
  |WHERE tag.tag_id IN
  |(
  |    SELECT tag_id
  |    FROM sidewalk.label_tag
  |    WHERE label_tag.label_id = ?
  |)"""
```

Or you can put `SELECT tag_id FROM sidewalk.label_tag WHERE label_tag.label_id = ?` on one line if the line is less than 100 characters long.